datasets/trac2_aggression/trac2_aggression.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow datasets Authors and the HuggingFace, DataLab Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+import os
+
+import pandas as pd
+
+# import datasets
+
+import datalabs
+from datalabs import get_task, TaskType
+
+_DESCRIPTION = """\
+Sub-task A: Aggression Identification Shared Task. The task will be to develop a classifier that could make a 3-way classification in between ‘Overtly Aggressive’, ‘Covertly Aggressive’ and ‘Non-aggressive’ text data. We are making available a dataset of 5,000 aggression-annotated data from social media each in Bangla (in both Roman and Bangla script), Hindi (in both Roman and Devanagari script) and English for training and validation. We will release additional data for testing your system. The train and test sets for the tasks are different from the ones made available during TRAC - 1.
+"""
+
+_HOMEPAGE = "https://docs.google.com/spreadsheets/d/1fZNZEi52i5GcsI2SqlTpRFC6OGUbhnJNSEDLlo1H624/edit#gid=0"
+
+
+_URL = "https://s3.amazonaws.com/datalab-hub/toxicity_detection/trac2_taskAB.zip"
+
+
+
+
+class TRAC2AggressionPred(datalabs.GeneratorBasedBuilder):
+    """This is a dataset of comments from Wikipedia’s talk page edits which have been labeled by human raters for toxic behavior."""
+
+    VERSION = datalabs.Version("1.1.0")
+
+    def _info(self):
+
+        return datalabs.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datalabs.Features(
+                {
+                    "text": datalabs.Value("string"),
+                    "label": datalabs.Value("string"),
+                }
+            ),
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            # license=_LICENSE,
+            task_templates=[get_task(TaskType.hatespeech_identification)(text_column="text", label_column="label")],
```

Comment:
maybe `toxicity_identification` would be more accurate? If yes, let's define it as a new task and use it.